packages/design-picker/src/components/premium-badge/index.tsx

```diff
@@ -19,7 +19,7 @@ const PremiumBadge: FunctionComponent< Props > = ( { className } ) => {
 			// @ts-expect-error: @types/wordpress__components doesn't align with latest @wordpress/components
 			delay={ 300 }
 			text={ __(
-				'Let your site stand out from the crowd with a modern and stylish Premium theme.',
+				'Let your site stand out from the crowd with a modern and stylish Premium theme. Premium themes are included in your plan.',
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.